### PR TITLE
Witness data in hotspot details

### LINKED ISSUE #143: Hotspot Details: Witnesses over time
Now that the bucketing api work is complete, we can show the witnesses over time.

### PR DESCRIPTION
**Pull request by tyler-whitman**
closes #143

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,6 +13,14 @@ if (__DEV__) {
   import('./ReactotronConfig')
 }
 
+// import { LogBox } from 'react-native'
```
Comment (tyler-whitman) [inline index.js:16]: I added this so you can uncomment to ignore the warnings from the bottom sheet

File: src/store/hotspotDetails/hotspotDetailsSlice.ts [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import { createAsyncThunk, createSlice } from '@reduxjs/toolkit'
 import { HotspotReward, HotspotRewardSum, Hotspot } from '@helium/http'
+import { WitnessSum } from '@helium/http/build/models/WitnessSum'
```
Comment (tyler-whitman) [inline src/store/hotspotDetails/hotspotDetailsSlice.ts:3]: I forgot to export this one in the index of helium-js, I'll make a PR to add that for the future

Comment (tyler-whitman) [reply src/store/hotspotDetails/hotspotDetailsSlice.ts:3]: https://github.com/helium/helium-js/pull/141

**Review by allenan — APPROVED** [review]
this works great, shipping it

**Pull request merged**